examples/scala/src/main/scala/example/Quickstart.scala

```diff
@@ -25,16 +26,20 @@ import java.io.File
 
 object Quickstart {
   def main(args: Array[String]): Unit = {
-    // Create a Spark Session
+
     val spark = SparkSession
       .builder()
-      .appName("Quickstart")
+      .appName("Streaming")
```

Comment:
I think I made this mistake, This should be Quickstart

Reply:
yess